From 39f7819cfe26f00921cb653a5d34974ce7af8d55 Mon Sep 17 00:00:00 2001
From: William Hubbs <w.d.hubbs@gmail.com>
Date: Sun, 27 Feb 2022 11:15:54 -0600
Subject: [PATCH aerc] fix desktop file

This was reported to me by a user of Gentoo (I'm a maintainer there) in
the following bug.

Bug: https://bugs.gentoo.org/834145
Signed-off-by: William Hubbs <w.d.hubbs@gmail.com>
---
 contrib/aerc.desktop | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/contrib/aerc.desktop b/contrib/aerc.desktop
index 235da5a..ddd43ae 100644
--- a/contrib/aerc.desktop
+++ b/contrib/aerc.desktop
@@ -13,6 +13,8 @@ Type=Application
 Icon=utilities-terminal
 Terminal=true
 Exec=aerc %u
+Actions=compose;
+MimeType=x-scheme-handler/mailto;
 
 [Desktop Action compose]
 Name=Compose mail
@@ -20,7 +22,4 @@ Name=Compose mail
 # enable as default mailto: handler using
 # xdg-mime default aerc.desktop x-scheme-handler/mailto
 Exec=aerc %u
-MimeType=x-scheme-handler/mailto;
-Type=Application
 Icon=utilities-terminal
-Terminal=true
-- 
2.34.1

